battlecode.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+mtput() {
+    if command -v tput > /dev/null; then
+        tput $@
+    fi
+}
+
+# Use tput to show different colors in the terminal
+mtput setaf 5
+echo "$ pip3 install --user cffi eel tqdm werkzeug ujson psutil"
+mtput sgr0
+pip3 install -q --user cffi eel tqdm werkzeug ujson psutil
```

Comment:
you can remove `eel` from here as well :)

Reply:
Isn't eel still required for the gui.py file? I suppose it is not strictly required for a cli though, but perhaps we will add a --web flag later.